wallet/src/Pos/Wallet/Web/Methods/Logic.hs

```diff
@@ -119,7 +119,7 @@ getAccountMod
     -> AccountId
     -> m CAccount
 getAccountMod balAndUtxo accMod accId = do
-    dbAddrs    <- getAccountAddrsOrThrow Existing accId
+    dbAddrs    <- getAccountAddrsOrThrow Existing (NeedSorting True) accId
```

Comment:
This is the only one that needs it! This is where we should use a version that returns the full data and does the sorting right here on the result. The full data will include the sorting key.

Reply:
@dcoutts I hope I have captured the spirit of your review: https://github.com/input-output-hk/cardano-sl/compare/alfredo/hotfix/csl2087-no-extra-sort...alfredo/hotfix/no-extra-sort-surgery?expand=1